zulip/client_test.go

```diff
@@ -151,22 +123,23 @@ func TestClient_PostToTopic(t *testing.T) {
 
 func TestClient_SendUserMessage(t *testing.T) {
 	srv := mockServer(t, func(w http.ResponseWriter, r *http.Request) {
-		assertEqual(t, r.Method, http.MethodPost)
-		assertEqual(t, r.URL.Path, "/messages")
+		assert.Equal(t, http.MethodPost, r.Method)
```

Comment:
These look flipped -- correct on the base, Yoda-like here?

Reply:
Oh that one was actually *right* lol